client/jetpack-cloud/sections/manage/pricing/controller.tsx

```diff
@@ -1,6 +1,8 @@
 import { type Callback } from '@automattic/calypso-router';
 import ManagePricingPage from 'calypso/jetpack-cloud/sections/manage/pricing/primary';
 
+export const JETPACK_COM_A4A_LANDING_PAGE = 'https://jetpack.com/for-agencies/';
```

Comment:
This seems like a weird place for a constant such as this, I see that there is no `constants.ts` file in this `manage/pricing` directory. Do you think we could add one so we have a good place for it?

Reply:
Yes, sounds good.  Thanks! 👍